### PR TITLE
Recommend badges

### PR DESCRIPTION
**Pull request by gaocegege**
Signed-off-by: Ce Gao <ce.gao@outlook.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,8 @@
 [![GoDoc](https://img.shields.io/badge/godoc-reference-blue.svg)](https://godoc.org/github.com/gaocegege/maintainer)
 [![Build Status](https://travis-ci.org/gaocegege/maintainer.svg?branch=master)](https://travis-ci.org/gaocegege/maintainer)
 [![](https://img.shields.io/badge/docker-supported-blue.svg)](https://hub.docker.com/r/gaocegege/maintainer/)
+[![Libraries.io for GitHub](https://img.shields.io/librariesio/github/gaocegege/maintainer.svg)](https://libraries.io/github/gaocegege/maintainer)
+[![Gitter](https://badges.gitter.im/gaocegege/maintainer.svg)](https://gitter.im/gaocegege/maintainer?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
```
Comment (at15) [inline README.md]: I was wondering why you didn't add gitter

Comment (gaocegege) [inline README.md]: All in this PR. I'm lazy

Comment (at15) [inline README.md]: no, you are pretty active when it comes to buying new stuff

Comment (gaocegege) [inline README.md]: 微笑

Comment (gaocegege) [inline README.md]: [LGTM](https://lgtm.in/i/iVuiCwbUp)

**Pull request closed without merging**